### PR TITLE
Gpl-378 create ont run

### LINKED ISSUE #297: GPL-378 As a user I need to be able to create an ONT run

### PR DESCRIPTION
**Pull request by harrietc52**
Closes #297

### CODE REVIEW

**Comment by harrietc52** [comment]
TODO:
move resolvers into component
move apollo.js file into graphql
reload runs on create
redirect shows alert message
refactor status in flowcell component
check if extra variable is required in gql queries

File: src/components/ont/OntFlowcell.vue [hunk 1]
```diff
@@ -1,10 +1,17 @@
 <template>
-  <g :transform="getMatrix">
-    <rect width="61" height="227"/>
+  <g :transform="getMatrix" v-on:drop="drop" v-on:dragover="allowDrop" >
+    <text x="25" y="30" class="medium">{{ position }}</text>
+
+    <rect width="61" height="227" v-bind:class="status"/>
+
+    <foreignObject y="100" width="70" height="227">
+      <b-form-input v-model="libraryName" placeholder="Name" :id="'libraryNameInput-'+this.position" @change="updateFlowcell"></b-form-input>
```
Comment (stevieing) [inline src/components/ont/OntFlowcell.vue:8]: It is difficult to see the full name of the library when it has been dropped onto the flowcell.

Comment (harrietc52) [reply src/components/ont/OntFlowcell.vue:9]: Made it a bit wierd but struggled with SVG HTML/CSS -> Added a tooltip as a quick fix

Comment (stevieing) [reply src/components/ont/OntFlowcell.vue:9]: ok. Lets add this issue to #421 . We can do a deep dive into the CSS to see what is going on. If we get some time next week we can have more of a look. I know how much of a pain CSS can be especially when it is wrapped up in bootstrap.

**Comment by stevieing** [comment]
> Is there any chance we could move the apollo.js file to the graphql folder just to keep it all in one place. As we start adding more logic to this it feels like a more logical place to put it.
> 
> UPDATE
> Looked into this, and I think an apollo.js config file is required at the root. I've refactored code of it into the graphql folder

Weird. I tried it in my branch and it was ok. No matter. It is pretty clean anyway.

**Pull request merged**